### ISSUE TITLE
Questions about mlm_acc and pretrain loss.

### ISSUE THREAD

**Issue by keqicmz** [935963004/LaBraM#64]
Hi authors,

Thank you for your excellent work! I have some problems during the **pre-training** phase of the training process. The loss did not show a significant decrease during the training process. The mlm_acc is very low. Have you encountered similar problems during the training process? And i want to know which aspect should the investigation be conducted?
Thank you very much.
```
{"train_lr": 2.4759998796232205e-05, "train_min_lr": 2.4759998796232205e-05, "train_mlm_acc": 0.004433143719016142, "train_mlm_acc_sym": 0.004432230200208095, "train_loss_rec": 3.851059658631664, "train_loss": 15.404179030606118, "train_loss_scale": 4068.1992709599026, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 0, "n_parameters": 29890080}
{"train_lr": 7.476150350597369e-05, "train_min_lr": 7.476150350597369e-05, "train_mlm_acc": 0.006605447829720951, "train_mlm_acc_sym": 0.006592208115589937, "train_loss_rec": 3.6628010799137383, "train_loss": 14.65092157738177, "train_loss_scale": 209.14702308626974, "train_weight_decay": 0.05000000000000097, "train_grad_norm": 29.84559377872205, "epoch": 1, "n_parameters": 29890080}
{"train_lr": 0.00012476300821571517, "train_min_lr": 0.00012476300821571517, "train_mlm_acc": 0.007509304827558984, "train_mlm_acc_sym": 0.007516131157550298, "train_loss_rec": 3.6341265639152156, "train_loss": 14.536196911030812, "train_loss_scale": 548.3936816524908, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 2, "n_parameters": 29890080}
{"train_lr": 0.00017476451292545666, "train_min_lr": 0.00017476451292545666, "train_mlm_acc": 0.008159913905981966, "train_mlm_acc_sym": 0.008155546220696168, "train_loss_rec": 3.621396214132031, "train_loss": 14.485185195435006, "train_loss_scale": 712.9428918590522, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 3, "n_parameters": 29890080}
{"train_lr": 0.0002247660176351982, "train_min_lr": 0.0002247660176351982, "train_mlm_acc": 0.008764938140967815, "train_mlm_acc_sym": 0.008767096283920573, "train_loss_rec": 3.611497012045085, "train_loss": 14.445247262824928, "train_loss_scale": 888.2235722964763, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 4, "n_parameters": 29890080}
{"train_lr": 0.00027476752234493963, "train_min_lr": 0.00027476752234493963, "train_mlm_acc": 0.009401158534852112, "train_mlm_acc_sym": 0.00941449039911892, "train_loss_rec": 3.60232360934429, "train_loss": 14.408688602441716, "train_loss_scale": 1199.4362089914946, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 5, "n_parameters": 29890080}
{"train_lr": 0.00032476902705468107, "train_min_lr": 0.00032476902705468107, "train_mlm_acc": 0.009855617207425782, "train_mlm_acc_sym": 0.009876932555236452, "train_loss_rec": 3.595562697108722, "train_loss": 14.38132633415491, "train_loss_scale": 1776.4471445929526, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 6, "n_parameters": 29890080}
{"train_lr": 7.476150350597369e-05, "train_min_lr": 7.476150350597369e-05, "train_mlm_acc": 0.010303858481672902, "train_mlm_acc_sym": 0.010313652842679035, "train_loss_rec": 3.588224585080929, "train_loss": 14.351995073286036, "train_loss_scale": 2309.910085054678, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 1, "n_parameters": 29890080}
{"train_lr": 0.00012476300821571517, "train_min_lr": 0.00012476300821571517, "train_mlm_acc": 0.010521738778942356, "train_mlm_acc_sym": 0.010530472653937771, "train_loss_rec": 3.584411618558923, "train_loss": 14.33702259842999, "train_loss_scale": 2851.771567436209, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 2, "n_parameters": 29890080}
{"train_lr": 0.00017476451292545666, "train_min_lr": 0.00017476451292545666, "train_mlm_acc": 0.010620267186237483, "train_mlm_acc_sym": 0.010637314312605874, "train_loss_rec": 3.582861111419024, "train_loss": 14.330433954879696, "train_loss_scale": 2855.5042527339006, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 3, "n_parameters": 29890080}
{"train_lr": 7.476150350597369e-05, "train_min_lr": 7.476150350597369e-05, "train_mlm_acc": 0.010809805400583006, "train_mlm_acc_sym": 0.01082174822990528, "train_loss_rec": 3.579844082147788, "train_loss": 14.318387821262402, "train_loss_scale": 2848.0388821385177, "train_weight_decay": 0.05000000000000097, "train_grad_norm": Infinity, "epoch": 1, "n_parameters": 29890080}
```
